kansha/board/comp.py

```diff
@@ -393,13 +388,20 @@ def set_archive(self, value):
         self.refresh()
         self.set_reload_search()
 
-    def archive_card(self, c):
+    def archive_card(self, cards, column):
         """Archive card
 
         In:
-            - ``c`` -- card to archive
+            - ``cards`` -- card to archive, or list of cards of same origin
+            - ``column`` -- origin column
         """
-        c.move_card(0, self.archive_column)
+        if not isinstance(cards, (list, tuple)):
+            cards = [cards]
+        for card in cards:
+            # only board can move cards, not card themselves, for they should have no idea of columns
+            notifications.add_event_to_history(self, ('card_archive', [card, None], None))
+            card.move_card(0, self.archive_column)
```

Comment:
I completely agree, that's the intent of issue #84 and others to come (for Column and Board).